source/tutorials/moving-pictures.rst

```diff
@@ -367,8 +367,7 @@ Next, we can view the taxonomic composition of our samples with interactive bar
 Differential abundance analysis
 -------------------------------
 
-Finally, we can quantify the process of identifying taxa that are differentially abundance (or present in different abundances) across sample groups. We do that using ANCOM (`Mandal et al. (2015)`_), which is implemented in the ``q2-composition`` plugin. ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on relative frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. We work around this by adding a pseudocount of 1 to every count in our ``FeatureTable[Frequency]`` table. We can run this on the ``BodySite`` category to determine what features differ in abundance across body sites. This step may take about 5 minutes to complete.
-
+Finally, we can quantify the process of identifying taxa that are differentially abundance (or present in different abundances) across sample groups. We do that using ANCOM (`Mandal et al. (2015)`_), which is implemented in the ``q2-composition`` plugin. ANCOM operates on a ``FeatureTable[Composition]`` QIIME 2 artifact, which is based on frequencies of features on a per-sample basis, but cannot tolerate frequencies of zero. To build the composition artifact, a ``FeatureTable[Frequency]``  artifact must be provided to add-pseudocount (an imputation method), which will then produce the ``FeatureTable[Composition]`` artifact. We can then run ANCOM on the ``BodySite`` category to determine what features differ in abundance across body sites. This step may take about 5 minutes to complete.
```

Comment:
Sorry just noticed a typo: in the first sentence, `differentially abundance` should be `differentially abundant`

Also I would drop the `then` from this fragment: `...(an imputation method), which will <then> produce...`